Config

```diff
@@ -1,26 +0,0 @@
-package.CredentialsFetcher = {
```

Comment:
Did you mean to delete this file?

Reply:
Yes, we meant to delete this file, it was a brazil file